### ISSUE TITLE
Incorrect tex file creation.

### ISSUE THREAD

**Issue by Arshtm** [jupyter/nbconvert#1522]
Hello! I use the Cadabra system in a Jupyter notebook and I have expressions like this as output of cells:
```
{}R_{a b} = \square{}_{a}{}_{b}\left\{\begin{aligned}\square{}_{t}{}_{t}& = -2\ddot{a} {a}^{-1}\\[-.5ex] \square{}_{x}{}_{x}& = a \ddot{a}+{\left(\dot{a}\right)}^{2}\\[-.5ex] \square{}_{y}{}_{y}& = a \ddot{a}+{\left(\dot{a}\right)}^{2}\\[-.5ex] \end{aligned}\right.
--
```
or after rendering:
![image](https://user-images.githubusercontent.com/53522332/108268780-14941880-717e-11eb-9cfe-4fbff2ca49d2.png)

But when I try to export the result as a latex file, it does not compile, because the same expression in it has the form:

```
 \$\{\}R\_\{a b\} = \square{}\emph{\{a\}\{\}}\{b\}\left\{

\begin{aligned}\square{}_{t}{}_{t}& = -2\ddot{a} {a}^{-1}\\[-.5ex]
\square{}_{x}{}_{x}& = a \ddot{a}+{\left(\dot{a}\right)}^{2}\\[-.5ex]
\square{}_{y}{}_{y}& = a \ddot{a}+{\left(\dot{a}\right)}^{2}\\[-.5ex]
\end{aligned}

\right. \$
```
I can handle each such expression separately so that it compiles, but maybe there is some other way out?
I would be glad for any help

**Comment by t-makaro** [comment]
Is there a space between the opening dollar sign and the first character in the expression? If so, this is a duplicate of #749.

**Comment by Arshtm** [comment]
> Is there a space between the opening dollar sign and the first character in the expression? If so, this is a duplicate of #749.

No, as you can see from the code at the bottom of my question, which I took from the generated file, the expression is surrounded by \ $ .. \ $, without spaces

**Comment by t-makaro** [comment]
What about in the notebook before conversion?

```
$E=mc^2$
```
converts fine, but
```
$ E=mc^2 $
```
fails because the ```$``` converts to ```\$```.

**Comment by Arshtm** [comment]
> What about in the notebook before conversion?
> 
> ```
> $E=mc^2$
> ```
> 
> converts fine, but
> 
> ```
> $ E=mc^2 $
> ```
> 
> fails because the `$` converts to `\$`.
> 
> If an output cell of your notebook has this problem, then the package that produces that output may need to be fixed.

No, as I wrote above, my output does not have any delimiters initially, as shown in the first piece of code in the question itself.
If I output it as plain text, it looks like this:
![image](https://user-images.githubusercontent.com/53522332/108591162-5d172600-7378-11eb-959c-a21206ea8edb.png)
